spec/policies/cart_item_policy_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CartItemPolicy, type: :policy do
+  let(:cart_item) { create(:cart_item) }
+  let(:user) { User.find(Cart.find(cart_item.cart_id).user_id) }
```

Comment:
maybe best way it's cart_item.cart.user_id? Not tested though

Reply:
check it

Reply:
User.find(cart_item.cart.user_id)